x-pack/test/functional/services/ml/ml_nodes_list.ts

```diff
@@ -25,11 +26,13 @@ export function MlNodesPanelProvider({ getService }: FtrProviderContext) {
     },
 
     async assertMlNodesCount(minCount: number = 1) {
-      const actualCount = parseInt(await testSubjects.getVisibleText('mlTotalNodesCount'), 10);
-      expect(actualCount).to.not.be.lessThan(
-        minCount,
-        `Total ML nodes count should be at least '${minCount}' (got '${actualCount}')`
-      );
+      await retry.try(async () => {
+        const actualCount = parseInt(await testSubjects.getVisibleText('mlTotalNodesCount'), 10);
+        expect(actualCount).to.not.be.lessThan(
+          minCount,
+          `Total ML nodes count should be at least '${minCount}' (got '${actualCount}')`
+        );
+      });
```

Comment:
@elastic/ml-ui

I encountered flakiness in some of the ml FTR test suite during this PR. After triple checking (well more like 10-checking) all API calls performed during the suite and manually reproducing the FTR scenario, I came to the conclusion that is was real FTR flakiness and not a bug introduced by this PR, so I had to add some retry behavior to some of your assertion utilities. Note that this is already what was done in most of them, only a few were missing.